ingestion/functions/parsing/riograndedosul/riograndedosul.py

```diff
@@ -0,0 +1,246 @@
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+_AGE_INDEX = 5
+_GENDER_INDEX = 4
+_ETHNICITY = 20
+_COUNTY = 1
+_DATE_CONFIRMED = 7
+_DATE_SYMPTOMS = 8
+_DATE_HOSPITALIZED = 11
+_DATE_DEATH = 18
+_FEVER = 12
+_COUGH = 13
+_SORETHROAT = 14
+_SHORTBREATH = 15
+_OTHER = 16
+_COMORBIDITIES = 17
+_INDIGENOUS = 21
+_NEIGHBORHOOD = 23
+
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+    """
+    date = datetime.strptime(raw_date, "%d/%m/%y")
+    return date.strftime("%m/%d/%YZ")
+
+
+def convert_gender(raw_gender: str):
+    if raw_gender == "Masculino":
+        return "Male"
+    if raw_gender == "Feminino":
+        return "Female"
+
+
+def convert_events(confirmed, symptoms, hospitalized, death):
+    events = [
+        {
+            "name": "confirmed",
+            "dateRange":
+            {
+                "start": convert_date(confirmed),
+                "end": convert_date(confirmed),
+            },
+        }
+    ]
+    if symptoms:
+        events.append({
+            "name": "onsetSymptoms",
+            "dateRange":
+            {
+                "start": convert_date(symptoms),
+                "end": convert_date(symptoms),
+            },
+        })
+    if hospitalized != "NAO":
+        events.append({
+            "name": "hospitalAdmission",
+            "dateRange":
+            {
+                "start": convert_date(hospitalized),
+                "end": convert_date(hospitalized),
+            },
+            "value": "Yes"
+        })
+    if death:
+        events.append({
+            "name": "outcome",
+            "dateRange":
+            {
+                "start": convert_date(death),
+                "end": convert_date(death),
+            },
+            "value": "Death"
+        })
+    return events
+
+
+def convert_symptoms(fever: str, cough: str, sorethroat: str, shortbreath: str, other: str):
+    symptoms = []
+    if fever == "SIM":
+        symptoms.append("Fever")
+    if cough == "SIM":
+        symptoms.append("Cough")
+    if sorethroat == "SIM":
+        symptoms.append("Sore Throat")
+    if shortbreath == "SIM":
+        symptoms.append("Shortness of Breath")
+    if other == "SIM":
+        symptoms.append("Other")
+    return symptoms
+
+
+def convert_preexisting_conditions(raw_commorbidities: str):
+    preexistingConditions = {}
+    if raw_commorbidities:
```

Comment:
up to you but I prefer to keep indented left as much as possible, that means checking for early returns first:

if not raw_commorbidities:
  return None
...